addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -70,6 +70,22 @@ $input-border-color: $gray-400;
         --o-wsale-card-thumb-fill-mode: cover;
     }
 
+    .o_wsale_category_link {
+        width: 8rem;
+    }
+
+    .o_wsale_category_ellipsis {
+        white-space: nowrap;
+        text-overflow: ellipsis;
+
+        @supports (-webkit-line-clamp: 2) {
+            white-space: initial;
+            display: -webkit-box;
+            -webkit-line-clamp: 2;
```

Comment:
trancating categories would make my german shopping experience a bit difficult 🥲  
![image](https://github.com/user-attachments/assets/c5833d0f-41ff-48ee-95e1-7990b614a811)

---

Less appealing , but more functional 😉 
ps: note the last guy on the right, you should address long words scenario as well 

![image](https://github.com/user-attachments/assets/b4969885-00bb-4760-8a51-d58f2c343640)